src/test/ui/macro-reexport-removed.stderr

```diff
@@ -0,0 +1,18 @@
+error[E0557]: feature has been removed
+  --> $DIR/macro-reexport-removed.rs:13:12
+   |
+LL | #![feature(macro_reexport)] //~ ERROR feature has been removed
```

Comment:
This should probably have some note about why the feature was removed and what should be used instead.